index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title>Usman Suleiman Portfolio</title>
+
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Poppins:regular,bold,500&display=swap"
+    />
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <header class="app-bar">
+      <a href="/" class="logo"
+        ><img src="images/logo.svg" alt="Home" width="76" height="20"
+      /></a>
+      <a href="#menu" class="tertiary-button"
+        ><img src="images/menu-icon.svg" alt="Menu" width="24" height="24"
+      /></a>
+    </header>
+    <article class="headline">
+      <h1 class="header-2 color-n800 mb-12">I’m Usman, Glad to see you!</h1>
+      <p class="body-2 color-n600 mb-12 drop-shadow">
+        I’m a software developer! I can help you build a product feature or
+        website. Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <footer>
+        <section>
+          <h6 class="body-2 bolder color-b300 mb-12 drop-shadow">
+            LET'S CONNECT
+          </h6>
```

Comment:
- [ ] This is a call to action, so instead of an h6, use a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed (see design)

Reply:
I can't find a design for such button in the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)